docs/reference/mapping/types/range.asciidoc

```diff
@@ -4,6 +4,13 @@
 <titleabbrev>Range</titleabbrev>
 ++++
 
+A field that represents a relation between a lower and upper bound, using the
```

Comment:
nit: might be just me, but I feel like the field doesn't represent the "relation" between the bounds but a continuous range of values between those bounds. The relations are then used to define and query the range. Maybe my understanding is off here, not a native speaker, sorry.